src/data/processing.py

```diff
@@ -3,6 +3,14 @@
 from imgaug.augmentables import Keypoint, KeypointsOnImage
 from src.utils.field_geometry_transf import get_zero_row_idx
 import os
+from src.config.constants import CLASSIFICATION
+
+
+def directory() -> str:
```

Comment:
Move this funciton in utils. Make a new python file if necessary. Return the absolute path.